utils/notes.ts

```diff
@@ -0,0 +1,88 @@
+import { CitationInlineStyleKind } from './citations';
```

Comment:
The logic in this file should be identical to what I removed from `usePubNotes()`. I factored it out so we can use it in the export pipeline as well.